server/internal/mv/toolset.go

```diff
@@ -136,6 +136,7 @@ func DescribeToolsetEntry(
 		promptTemplates = append(promptTemplates, &types.PromptTemplateEntry{
 			ID:   pt.ID.String(),
 			Name: types.Slug(pt.Name),
+			Kind: &pt.Kind.String,
```

Comment:
What if this evaluates to `""`? Did you want to use `conv.FromPGText` and `conv.Default`?